var/spack/repos/builtin/packages/abinit/package.py

```diff
@@ -151,31 +144,28 @@ def install(self, spec, prefix):
         oapp("--with-dft-flavor=atompaw+libxc")
 
         # LibXC library
+        libxc = spec["libxc:fortran"]
         options.extend([
-            "with_libxc_incs=-I%s" % spec["libxc"].prefix.include,
-            "with_libxc_libs=-L%s -lxcf90 -lxc" % spec["libxc"].prefix.lib,
+            "with_libxc_incs={0}".format(libxc.cppflags),
+            "with_libxc_libs={0}".format(libxc.libs.ld_flags + ' -lm')
```

Comment:
I don't think `libm` should go in `libxc` (see [here](http://stackoverflow.com/questions/1033898/why-do-you-have-to-link-the-math-library-in-c) for a list of reasons why) but, given that in the `builtin` repository it appears 26 times over 16 files, maybe we need a better way to refer to it?